news/catout.rst

```diff
@@ -0,0 +1,12 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+* cat from xoreutils now outputs in configured encoding
```

Comment:
Would you mind adding a newline between these two, please? The tests are failing because this is not valid rst Thanks!